pkg/ebpf/tracee.go

```diff
@@ -1336,6 +1339,7 @@ func (t *Tracee) triggerSeqOpsIntegrityCheck(event trace.Event) {
 //go:noinline
 func (t *Tracee) triggerSeqOpsIntegrityCheckCall(
 	magicNumber uint64, // 1st arg: allow handler to detect calling convention
+	traceePid uint64, // 2nd arg: used to determine if relevant to current tracee instance
```

Comment:
pid is u32, maybe we could use a single argument for both (higher bits: a magic number, lower bits: the pid) BUT this is not needed for this PR, more of a mental note for all of us if we need more than 3 arguments (I believe eBPF ISA has a max of 5 arguments for function arguments).